test/y2storage/boot_requirements_checker_x86_test.rb

```diff
@@ -290,14 +290,31 @@
       end
 
       context "when proposing a new EFI partition" do
+
         let(:efi_part) { find_vol("/boot/efi", checker.needed_partitions(target)) }
         let(:desired_efi_part) { find_vol("/boot/efi", checker.needed_partitions(:desired)) }
         # Default values to ensure proposal of EFI partition
         let(:efiboot) { true }
         let(:efi_partitions) { [] }
 
         include_examples "proposed EFI partition basics"
-        include_examples "flexible size EFI partition"
+
+        context "and BLS bootloader as default" do
```

Comment:
To make the document more understandable I would probably use something like: "if BLS installation is not explicitly disabled". Or would simply skip this whole context and add the `include_examples` directly, since this is now the default behavior.

Reply:
Hm, not sure if I have done the right thing now. I have renamed it at least.